src/index.ts

```diff
@@ -246,6 +246,7 @@ export * as ucan from './ucan'
 export * as dns from './dns'
 export * as ipfs from './ipfs'
 export * as keystore from './keystore'
+export * as common from './common'
```

Comment:
What's the reason you've added this @dholms?

Reply:
I wanted to use one of the common methods (`makeUrlUnsafe`) in the tests instead of rewriting logic, and didn't see any harm in exporting the methods. But can go either way with it honeslty :man_shrugging:

Reply:
Ah gotcha 👍  I'll rename this to something else, because `common` doesn't make much sense from the user's perspective.

Reply:
Renamed to `webnative.machinery`

Reply:
love it